chapter_linear-networks/fashion-mnist_vn.md

```diff
@@ -261,7 +264,10 @@ We are now ready to work with the FashionMNIST dataset in the sections that foll
 4. Use the MXNet documentation to see which other transformations are available in `mxnet.gluon.data.vision.transforms`.
 -->
 
-*dịch đoạn phía trên*
+1. Việc giảm `batch_size` (ví dụ xuống 1) có ảnh hưởng việc đọc dữ liệu hay không?
+2. Với các người dùng không sử dụng Windows, hãy thử thay đổi `num_workers` để xem nó ảnh hưởng đến hiệu năng đọc dữ liệu như thế nào. Vẽ đồ thị hiệu năng tương ứng với số luồng được sử dụng.
```

Comment:
```suggestion
2. Với các người dùng không sử dụng Windows, hãy thử thay đổi `num_workers` để xem nó ảnh hưởng đến chất lượng đọc dữ liệu như thế nào. Vẽ đồ thị chất lượng tương ứng với số luồng được sử dụng.
```

Reply:
@duythanhvn về tính toán thì `performance` lại là `hiệu năng` bạn trẻ ạ.

Reply:
Chắc phải note lại bối cảnh trong glossary cho từ `performance`

Reply:
My bad!